treeherder/model/management/commands/cycle_data.py

```diff
@@ -1,52 +1,53 @@
 import datetime
+from optparse import make_option
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
 
-from treeherder.model.derived import JobsModel
-from treeherder.model.models import (Datasource,
-                                     Job,
+from treeherder.model.models import (Job,
                                      JobGroup,
                                      JobType,
-                                     Machine)
+                                     Machine,
+                                     Repository)
 
 
 class Command(BaseCommand):
     help = """Cycle data that exceeds the time constraint limit"""
 
-    def add_arguments(self, parser):
-        parser.add_argument(
+    option_list = BaseCommand.option_list + (
```

Comment:
The `option_list` method of specifying arguments was deprecated in Django 1.8, and removed entirely in Django 1.10 - could you leave it as the newer argparse  `add_argument()` form?